docs/installation-and-operations/configuration/README.md

```diff
@@ -8,19 +8,155 @@ sidebar_navigation:
 
 
 
-OpenProject can be configured either via the `config/configuration.yml` file, [environment variables](environment/) or a mix of both.
-While the latter is probably a bad idea, the environment variable option is often helpful for automatically deploying production systems.
-Using the configuration file is probably the simplest way of configuration.
+OpenProject can be configured either via environment variables. These are often helpful for automatically deploying production systems.
 
-You can find a list of options below and an example file in [config/configuration.yml.example](https://github.com/opf/openproject/blob/dev/config/configuration.yml.example) 
+> **NOTE:** This documentation is for OpenProject on-premises Installations only, if you would like to setup similar in your OpenProject cloud instance, please contact us at support@openproject.com
 
+> **NOTE:** Using the configuration file `config/configuration.yml` is depracted and is **NOT** recommended anymore
 
 
-## Environment variables
 
-Configuring OpenProject through environment variables is detailed [in this separate guide](environment/).
+# Packaged installation
 
-## List of options
+The file `/opt/openproject/.env.example` contains some information to learn more. The file `/opt/openproject/conf.d/env` is used for parsing the variables and your custom values to your configuration.
+
+To configure the environment variables such as the number of web server threads OPENPROJECT_HTTPS, copy the `.env.example` to `/etc/openproject/conf.d/env` and add the environment variables you want to configure. The variables will be automatically loaded to the application’s environment.
+
+After changing the file `/etc/openproject/conf.d/env`  the command `sudo openproject configure` must be issued
+
+If you would like to change a value of a variable without a downtime of OpenProject you are able to configure the environment variable by using the command:
```

Comment:
This is not correct. Using `openproject config:set ...` has the same effect as putting it to `/etc/openproject/conf.d/other` and still requires a restart of the server

Reply:
thank you for clarifying this Oliver I will change the text

Reply:
@oliverguenther i changed your remark and continued in editing the configuration files, this PR will change nearly everything to be vonfigured in via ENV, just one thing i will not be able to do is the repository page, so i changed that to being outdated in a NOTE on the first row